src/main/java/cl/transbank/patpass/PatpassComercio.java

```diff
@@ -80,16 +80,15 @@ public static PatpassComercioInscriptionStartResponse start(String url,
                                                                     String rut,
                                                                     String serviceId,
                                                                     String finalUrl,
-                                                                    String commerceCode,
-                                                                    double maxAmount,
+                                                                    String maxAmount,
```

Comment:
Why did you have to change it to String instead of Double?

Reply:
cause, can't we have an empty Double, can we?

Reply:
You can always use `null`. Why can not you?

Reply:
changed to double and added new logic to handle empty or value equal zero for maxAmount